modules/apps/web-experience/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/page.jsp

```diff
@@ -67,6 +68,11 @@ String tagNames = GetterUtil.getString((String)request.getAttribute("liferay-ass
 		}
 	).render();
 
+	var assetTagNamesTextArea = document.getElementsByName('assetTagNames');
+
+	assetTagNamesTextArea[0].setAttribute('style', 'height:' + (assetTagNamesTextArea[0].scrollHeight) + 'px;overflow-y:hidden;');
+	assetTagNamesTextArea[0].addEventListener("input", assetTaglibTagsSelector._onInput, false);
```

Comment:
We should not be a calling a protected method.  Protected methods should only be called from within the JS file itself and not externally in case the API for this method changes and breaks everything.

Reply:
This does not seem right since it will only grab the first field named ```assetTagNames```.  If there are 2 fields named```assetTagNames```, this will only update the first one.